scripts/inboxtable.js

```diff
@@ -9,13 +9,6 @@
         $.fn.dataTableExt.oStdClasses.sStripeOdd = "row r0";
         $.fn.dataTableExt.oStdClasses.sStripeEven = "row r1";
 
-        // Decide table ordering based on langconfig setting
-        if (date_format ==  "%d/%m/%y, %H:%M") {
```

Comment:
If you are deleting date_format, remove the parameter from the init function above. There will also be a place where the datatable is initialised that passes in date format. As well as that, you probably want to delete the custom sort filter files if we aren't using them any more, just to tidy everything up.